src/util/redis/redisConnector.js

```diff
@@ -77,20 +77,12 @@ const RedisDB = {
    * key2: {internalKey2:val2},
    * }
    */
-  async getVal(hashKey, key, isObjExpected = true) {
+  async getVal(hashKey, isObjExpected = true) {
     try {
       await this.checkAndConnectConnection(); // check if redis is connected and if not, connect
       let redisVal;
       if (isObjExpected === true) {
-        redisVal = await this.client.hget(hashKey, key);
-        if (isDefinedAndNotNull(redisVal)) {
-          try {
-            return JSON.parse(redisVal);
-          } catch (e) {
-            // do nothing
-            return redisVal;
-          }
-        }
```

Comment:
Any reason we removed this parsing?

Reply:
We are now fetching all the data in one go and since it is stored in object format in redis itself so we don't need it to parse it again